Force the date of the last produced file to the newest package source file. 
Take care or receipt *and* stuff
--- a/cook
+++ b/cook
@@ -879,7 +879,8 @@
 
 	# Build cpio archives.
 	action 'Compressing the FS...'
-	find fs -newer $receipt -exec touch -hr $receipt {} \;
+	newest="$(find receipt stuff 2>/dev/null | xargs ls -td | sed q)"
+	find fs -newer $newest -exec touch -hr $newest {} \;
 	find fs | cpio -o -H newc --quiet | lzma e fs.cpio.lzma -si
 	rm -rf fs
 	status
